components/grade-table.js

```diff
@@ -1,25 +1,52 @@
 class GradeTable {
-    constructor(tableElement){
+    constructor(tableElement,noGradesElement){
         this.tableElement = tableElement;
+        this.noGradesElement = noGradesElement;
     }
+
     updateGrades(grades){
-        console.log('grades: ',grades);
         var tbody = this.tableElement.querySelector('tbody');
         tbody.textContent = '';
+        this.renderGradeRow(grades,this.deleteGrade);
+        var p = this.noGradesElement;
+        if(grades.length > 0){
+             p.className += ' d-none';
+        }
+        else  {
+             p.classList.remove('d-none');
+        }

```

Comment:
While this is acceptable, there is in-fact a `.classList.add(...)` that does the reverse of `.classList.remove(...)` that you may want to use in the future to make your code easier to read.